amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,967 @@ input AMPLIFY {
+  id: ID! @index(name: "byAddress", queryField: "getColonyByAddress")
+  """(Short) name of the Colony."""
   name: String! @index(name: "byName", queryField: "getColonyByName")
+  """The native token of the Colony."""
   nativeToken: Token! @hasOne
+  """List of tokens that are used within the Colony."""
   tokens: [Token] @manyToMany(relationName: "ColonyTokens")
+  """Status information for the Colony."""
   status: ColonyStatus
+  """List of domains of the Colony."""
   domains: [Domain] @hasMany(indexName: "byColony", fields: ["id"])
-  watchers: [User] @manyToMany(relationName: "WatchedColonies") # colony subscriptions
+  """List of users watching the Colony."""
+  watchers: [User] @manyToMany(relationName: "WatchedColonies")
+  """List of Colony funds claims for all ERC20 tokens."""
   # NOTE: Could not merge these two fields properly
   # Ideally we would merge data from these two into one field, but I couldn't do that
   # meaning we'll have to merge this data in-app (or not at all, works either way)
-  # If you have a better idea, on how to merged them, I'll all ears...
-  fundsClaims: [ColonyFundsClaim] @hasMany # All ERC20 token claims
-  # Native chain token claims (eg: Token 0x0000...0000)
+  # If you have a better idea, on how to merged them, I'll all ears..
+  fundsClaims: [ColonyFundsClaim] @hasMany
+  """
+  List of native chain token claims (e.g., Token 0x0000...0000: ETH, xDAI, etc.).
+  """
   # This is not an array since only a single token type can be returned
   chainFundsClaim: ColonyChainFundsClaim
     @function(name: "fetchColonyNativeFundsClaim")
+  """Type of the Colony (Regular or Metacolony)."""
   type: ColonyType @index(name: "byType", queryField: "getColonyByType")
-  # Returns a list token balances for each domain and each token that the colony has
+  """
+  List of token balances for each domain and each token that the Colony has.
+  """
   balances: ColonyBalances @function(name: "fetchColonyBalances")
+  """Metadata related to the chain of the Colony."""
   chainMetadata: ChainMetadata
+  """List of extensions installed in the Colony."""
   extensions: [ColonyExtension!] @hasMany(indexName: "byColony", fields: ["id"])
+  """Version of the Colony."""
   version: Int!
+  """List of actions that happened within the Colony."""
   actions: [ColonyAction] @hasMany
+  """Metadata of the Colony."""
   metadata: ColonyMetadata @hasOne(fields: ["id"])
 }
 
+"""
+Represents metadata for a Colony.
+"""
 type ColonyMetadata @model {
-  id: ID! # colony contract address
+  """Unique identifier for the Colony (contract address)."""
+  id: ID!
+  """Display name of the Colony."""
   displayName: String!
+  """URL of the Colony's avatar image."""
   avatar: String
+  """URL of the Colony's thumbnail image."""
   thumbnail: String
+  """List of Colony metadata changelog entries."""
   changelog: [ColonyMetadataChangelog!]
 }
 
+"""
+Represents a changelog entry for Colony metadata.
```

Comment:
This could mention it's used to traverse through the history of metadata values and compare how they changed